src/main/java/core/basesyntax/InterfacePaint.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface InterfacePaint {
+    void paintFigure();
```

Comment:
Our task is to **draw** - not to **print**

```suggestion
public interface Drawable{
    void draw();
```